www/js/search.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable arrow-parens */
+const { CONSTS, query, getAng, loadAng, getShabad, loadShabad } = require('./banidb');
```

Comment:
Interestingly, when you paste a query into the search box, it doesn't auto search. Not something you have to fix in this PR.

Reply:
https://github.com/KhalisFoundation/sttm-desktop/issues/316